tests/spdx/parser/jsonlikedict/test_dict_parsing_functions.py

```diff
@@ -1,13 +1,6 @@
-#  Copyright (c) 2022 spdx contributors
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#    http://www.apache.org/licenses/LICENSE-2.0
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
+# SPDX-FileCopyrightText: 2022 spdx contributors
+
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2022 spdx contributors
#
# SPDX-License-Identifier: Apache-2.0
```
I don't think that this is additional hastag is necessary but it would make these changes concsistent.

Reply:
done, needs another approval ;)

Reply:
this does also apply to the other test files :innocent:

Reply:
...and also to almost all other files, apparently^^
Have look again ;)